src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -60,6 +74,11 @@ const QuickMarcEditor = ({ instance, onClose, handleSubmit, initialValues, submi
               <QuickMarcEditorRows
                 fields={records}
                 name="records"
+                mutators={mutators}
+              />
+              <FormSpy
```

Comment:
why do you ignore comments? previously I described structure

Reply:
I noticed it after the last commit, it will be changed in the next one)